### PR TITLE
#157731618 Allow comparison of members

### PR DESCRIPTION
**Pull request by tibetegya**
### What does this PR do?
Allows comparison of members in case they are training together where by you have a graph where the data for (selected) members is shown .
#### Description of Task to be completed?
1. Run Autopep8 on code base
2. add local seetings import to settings.py 
#### How should this be manually tested?
1. Run the commands below
```
$ git clone https://github.com/andela/wg-rogue-one.git

$ git checkout ft-allow-comparison-157731618

```
follow the setup instructions in [README.rst](https://github.com/andela/wg-rogue-one/blob/ch-heroku-deployment-157842572/README.rst)
2. Run the app with `python manage.py runserver`
3. Login to wger as a `Trainer` or `Gym Manager` 
4. Navigate to the `Gym overview` and select two users.
screen shot 2018-06-06 at 12 52 47
5. Press the `compare` Button in-order to compare them. 

#### Any background context you want to provide?
When two users are training together it would be useful to see a comparison of some sorts. A possibility would be to have a graph where the data for (selected) members is shown or, less nicely, at least have different graphs under each other.
 
#### What are the relevant pivotal tracker stories?
#157731618

### CODE REVIEW

**Review by emugaya — CHANGES_REQUESTED** [review]
@tibetegya Your commit message says nothing about allowing comparison of members. The PR should have how I test the feature when logged in,  not when I have pulled your code.

**Comment by emugaya** [comment]
I see only `autopep8` related changes. Where is the code for comparing users?

**Comment by tibetegya** [comment]
@emugaya i first ran autopep8 commands before starting on the task but i have updated the pr instructions to show how to test it

**Comment by emugaya** [comment]
@tibetegya Please add the charts to the PR.

File: wger/gym/templates/gym/partial_user_list.html [hunk 1]
```diff
@@ -44,17 +54,17 @@
...
-                {% if show_gym %}
+                <!-- {% if show_gym %} -->
                 <td>
                     {% if current_user.obj.userprofile.gym_id %}
                         <a href="{{ current_user.obj.userprofile.gym.get_absolute_url }}">
                         {{ current_user.obj.userprofile.gym }}
                         </a>
-                    {% else %}
-                        -/-
+                    <!-- {% else %}
```
Comment (emugaya) [inline wger/gym/templates/gym/partial_user_list.html:63]: Remove commented code.

**Review by emugaya — APPROVED** [review]

**Pull request merged**